emm-docs/content/subcommands/pull-request.md

```diff
@@ -31,3 +31,7 @@ you must also provide a `--title`.
 ```bash
 $ evg-module-manager pull-request --title "my pull request title" --body "my pull request body"
 ```
+
+Note: Before using this command, you may want to confirm your git remote url. The
```

Comment:
Can we make this a callout so it will visually stand out for users? You can see an example of that in lines 6 - 12 of this file.